spec/controllers/sessions_controller_spec.rb

```diff
@@ -29,10 +29,46 @@
   end
 
   context 'GET :destroy' do
-    it 'destroys a user session' do
+    it 'destroys a user session', :aggregate_failures do
       get :destroy
       expect(response).to redirect_to(root_path)
       expect(flash[:notice]).not_to be_nil
     end
   end
+
+  context 'for email auth' do
+    before do
+      request.env['omniauth.auth'] = EMAIL_AUTH_HASH
+      request.host = 'hamburg.onruby.de'
+    end
+
+    context 'POST :email_login' do
+      it 'sends the email and redirect to index', :aggregate_failures do
+        expect { post :email_login, params: { email: 'user@somewhere.org' } }
```

Comment:
```suggestion
        expect { post :email_login, params: { email: 'user@example.org' } }
```
Somewhere.org is registered by "New Radio and Performing Arts, Inc.", that isn't a domain we should use. :) 
http://example.org